cachito/web/splash.py

```diff
@@ -2,7 +2,7 @@
 from flask import render_template, Blueprint
 
 
-splash = Blueprint('splash', __name__, static_folder='static', template_folder='template')
```

Comment:
Oh interesting! Not needed because it defaults to those values anyways?

Reply:
Yup, it works just fine.